front/app/components/SignUpIn/SignUp/Consent.tsx

```diff
@@ -92,36 +95,18 @@ const Consent = memo(
             <FormattedMessage {...messages.viennaConsentFooter} />
           </ConsentText>
 
-          <Box id="vienna-terms-and-conditions">
-            <Checkbox
-              size="20px"
-              checked={termsAndConditionsAccepted}
-              onChange={handleTermsAndConditionsOnChange}
-              label={
-                <ConsentText>
-                  <FormattedMessage
-                    {...messages.iHaveReadAndAgreeToVienna}
-                    values={{
-                      link: (
-                        <Link target="_blank" to="/pages/terms-and-conditions">
-                          <FormattedMessage
-                            {...messages.theTermsAndConditions}
-                          />
-                        </Link>
-                      ),
-                    }}
-                  />
-                </ConsentText>
-              }
-            />
-            <Error
-              text={
-                termsAndConditionsError
-                  ? formatMessage(messages.tacError)
-                  : null
-              }
+          <ConsentText>
+            <FormattedMessage
```

Comment:
This text must be bold, what's the easiest way for me to add this?

`strong` and `b` tags don't do it.

Reply:
You can use our `Text` component for ConsentText that has a bold prop available our of the box. Additional advantage - it's not semantic to use divs for text (as is currently the case for `ConsentText` - bad for accessibility